app/src/main/java/com/abhilekh/assignmentapplication/viewmodel/ListViewModel.kt

```diff
@@ -25,6 +25,8 @@ class ListViewModel(application: Application) : AndroidViewModel(application) {
     fun refresh() {
         Log.d("mvvm","viewmodel-> refresh")
         animals.postValue(Resource.loading(null))
+
+        println("")
```

Comment:
Please use Log instead java methods to print logs

Reply:
resolved #